sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -137,7 +138,7 @@
 
     return value
 
-
+# pylint: disable=too-many-return-statements
```

Comment:
lol.  how about we just disable that rule entirely.

Reply:
haha, we certainly can 😆